Resources/doc/resource_owners/fiware.md

```diff
@@ -0,0 +1,35 @@
+Step 2x: Setup FI-WARE
+=========================
+First you will have to register your application on your FI-WARE Identity Management - KeyRock. Check out the
```

Comment:
Same here, wrong in all other docs (facebook.md, foursquare.md, github.md etc.).

I think, someone who has the permission to commit, should fix all docs at once. That's much easier and consistent.